src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -25,18 +25,18 @@
 //
 //------------------------------------------------------------------------------
 
-using Microsoft.IdentityModel.Logging;
-using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
+using Microsoft.IdentityModel.Logging;
+using Newtonsoft.Json;
 
-namespace System.IdentityModel.Tokens.Jwt
+namespace Microsoft.IdentityModel.Tokens
 {
     /// <summary>
     /// Represents a Json Web Key as defined in http://tools.ietf.org/html/rfc7517.
     /// </summary>
     [JsonObject]
-    public class JsonWebKey
+    public class JsonWebKey : AsymmetricSecurityKey
```

Comment:
> Do you think retrieving symmetric JWK from metadata endpoint is a common scenario?

Nah, it's not a common scenario (it would probably have to require client authentication to avoid leaking the key material). But the real question is: why do you want to make `JsonWebKey` inherit from `SecurityKey`?